src/components/Button/index.js

```diff
@@ -1,8 +1,8 @@
 import './Button.scss'
 
-const Button = ({ name }) => {
+const Button = ({name, buttonSetter, id}) => {
```

Comment:
I now suggest renaming this component from button - it's now a button specific to the homepage, it will only really work for displaying a link to a restaurant - so calling it something like RestaurantButton would be more appropriate